changelogs/CHANGELOG-v0.5.4.md

```diff
@@ -0,0 +1,10 @@
+The v0.5.4 release is a patch release that fixes a libc incompatibility between
```

Comment:
```suggestion
The v0.5.4 release is a patch release that fixes a glibc incompatibility between
```

Reply:
>Also, do we want to update the v0.5.2 and v0.5.3 release notes with a statement on why not to use them?

I'll give it a try.

Reply:
glibc change made.  I also added a "Known Issues" section for v0.5.2 and v0.5.3.